pack.go

```diff
@@ -392,3 +423,11 @@ func ensureAnnotationCreated(annotations map[string]string, annotationCreatedKey
 	copied[annotationCreatedKey] = now.Format(time.RFC3339)
 	return copied, nil
 }
+
+// validateMediaType validates the format of mediaType.
+func validateMediaType(mediaType string) error {
+	if !mediaTypeRegexp.MatchString(mediaType) {
+		return errdef.ErrInvalidMediaType
```

Comment:
It seems can be refactored as 
```suggestion
		return fmt.Errorf("%s: %w", mediaType, errdef.ErrInvalidMediaType)
```